EuroPythonBot/extensions/programme_notifications/__init__.py

```diff
@@ -15,6 +15,9 @@
 
 _logger = logging.getLogger(f"bot.{__name__}")
 
+# timestamp during the conference used for testing (see timewarm in config)
+TIME_DURING_CONFERENCE = "2024-04-23T15:57:45+02:00"
```

Comment:
```suggestion
TIME_DURING_CONFERENCE = "2024-04-23T15:54:45+02:00"
```

Reply:
if you look in the config you have a 5 min time before the notification but only 2 for the room. [code](https://github.com/PYCONDE/discord/blob/program-notifications-extension/EuroPythonBot/extensions/programme_notifications/services/notifier.py#L16-L17)
however in the fake time you use for the conference, you set it to 2 min and 15 seconds before the next talks [code](https://github.com/PYCONDE/discord/blob/program-notifications-extension/EuroPythonBot/extensions/programme_notifications/__init__.py#L19).